pkgs/tools/security/vulnix/requirements.nix

```diff
@@ -0,0 +1,531 @@
+      sha256 = "b0bbe6dfe60e2bdcf842a87abcc53b5cd15ad88ae2824c0c526cbb01ca0b90e5";
+    };
+    doCheck = commonDoCheck;
+    buildInputs = commonBuildInputs;
+    propagatedBuildInputs = [
+      BTrees
+      ZConfig
+      persistent
+      six
+      transaction
+      zc.lockfile
+      zodbpickle
+      zope.interface
+    ];
+    meta = with stdenv.lib; {
+      homepage = "";
+      license = licenses.zpt21;
+      description = "Zope Object Database: object database and persistence";
+    };
+  };
+
+
+
+  click = buildPythonPackage {
+    name = "click-6.6";
+    src = fetchurl {
+      url = "https://pypi.python.org/packages/7a/00/c14926d8232b36b08218067bcd5853caefb4737cda3f0a47437151344792/click-6.6.tar.gz";
+      sha256 = "cc6a19da8ebff6e7074f731447ef7e112bd23adf3de5c597cf9989f2fd8defe9";
+    };
+    doCheck = commonDoCheck;
+    buildInputs = commonBuildInputs;
+    propagatedBuildInputs = [ ];
+    meta = with stdenv.lib; {
+      homepage = "";
+      license = licenses.bsdOriginal;
+      description = "A simple wrapper around optparse for powerful command line utilities.";
+    };
+  };
+
+
+
+  colorama = buildPythonPackage {
+    name = "colorama-0.3.7";
+    src = fetchurl {
+      url = "https://pypi.python.org/packages/f0/d0/21c6449df0ca9da74859edc40208b3a57df9aca7323118c913e58d442030/colorama-0.3.7.tar.gz";
+      sha256 = "e043c8d32527607223652021ff648fbb394d5e19cba9f1a698670b338c9d782b";
+    };
+    doCheck = commonDoCheck;
+    buildInputs = commonBuildInputs;
+    propagatedBuildInputs = [ ];
+    meta = with stdenv.lib; {
+      homepage = "";
+      license = licenses.bsdOriginal;
+      description = "Cross-platform colored terminal text.";
+    };
+  };
+
+
+
+  coverage = buildPythonPackage {
+    name = "coverage-4.0.3";
+    src = fetchurl {
+      url = "https://pypi.python.org/packages/76/b4/3777a6bae434240b1fcbbda6cb30085bd897b3519acfffea498ee9f41038/coverage-4.0.3.tar.gz";
+      sha256 = "85b1275b6d7a61ccc8024a4e9a4c9e896394776edce1a5d075ec116f91925462";
+    };
+    doCheck = commonDoCheck;
+    buildInputs = commonBuildInputs;
+    propagatedBuildInputs = [ ];
+    meta = with stdenv.lib; {
+      homepage = "";
+      license = licenses.asl20;
+      description = "Code coverage measurement for Python";
+    };
+  };
+
+
+
+  flake8 = buildPythonPackage {
+    name = "flake8-2.5.4";
+    src = fetchurl {
+      url = "https://pypi.python.org/packages/60/4a/7b0ac4920af5673380b7079ba2f7580a8645790c7718881082c0d918b8b4/flake8-2.5.4.tar.gz";
+      sha256 = "cc1e58179f6cf10524c7bfdd378f5536d0a61497688517791639a5ecc867492f";
+    };
+    doCheck = commonDoCheck;
+    buildInputs = commonBuildInputs;
+    propagatedBuildInputs = [
+      "mccabe"
+      "pep8"
+      "pyflakes"
+    ];
+    meta = with stdenv.lib; {
+      homepage = "";
+      license = licenses.mit;
+      description = "the modular source code checker: pep8, pyflakes and co";
+    };
+  };
+
+
+
+  lxml = buildPythonPackage {
```

Comment:
We have the habit to pin down versions of dependent packages. 

setup.py:
```
install_requires=[
        'click==6.6',
        'colorama==0.3.7',
        'lxml==3.7.0',
        'pyyaml==3.11',
        'requests==2.10.0',
        'ZODB==5.1.1',
    ],
```
If there are recommendations to override those in setup.py, I'd like to know them. I am really uncertain about the pattern of dependence on external packages vs. nixpkgs' ones.